modules/centreon-stream-connectors-lib/sc_event.lua

```diff
@@ -446,91 +446,133 @@ end
 function ScEvent:is_valid_hostgroup()
   self.event.cache.hostgroups = self.sc_broker:get_hostgroups(self.event.host_id)
 
-  -- return true if option is not set
-  if self.params.accepted_hostgroups == "" then
+  -- return true if options are not set or if both options are set
+  local accepted_hostgroups_isnotempty = self.params.accepted_hostgroups ~= ""
+  local rejected_hostgroups_isnotempty = self.params.rejected_hostgroups ~= ""
+  if (not accepted_hostgroups_isnotempty and not rejected_hostgroups_isnotempty) or (accepted_hostgroups_isnotempty and rejected_hostgroups_isnotempty) then
```

Comment:
I'm not sure but you are checking if : 
- only accepted hostgroup is set
- accepted host group and rejected host group are set together
but not if only rejected host group is set ?
Don't user may needs to only set rejected host group ?